mysql-test/suite/innodb/t/innodb_buffer_pool_resize_mdev_27891.test

```diff
@@ -0,0 +1,30 @@
+--source include/have_innodb.inc
+
+SET @save_size=@@GLOBAL.innodb_buffer_pool_size;
+
+let $wait_timeout = 60;
+let $wait_condition =
+  SELECT SUBSTR(variable_value, 1, 31) = 'Completed resizing buffer pool.'
+  FROM information_schema.global_status
+  WHERE variable_name = 'INNODB_BUFFER_POOL_RESIZE_STATUS';
+
+--echo
+--echo MDEV-27891: Delayed SIGSEGV in InnoDB buffer pool resize
+--echo after or during DROP TABLE
+--echo
+
+select @@innodb_buffer_pool_chunk_size;
+CREATE TABLE t1 (a INT PRIMARY KEY) ENGINE=InnoDB;
+SET GLOBAL innodb_buffer_pool_size=200*1024*1024;
+DROP TABLE t1;
+--error 0,ER_WRONG_ARGUMENTS
+SET GLOBAL innodb_buffer_pool_size=@@innodb_buffer_pool_size + @@innodb_buffer_pool_chunk_size;
+--source include/wait_condition.inc
+
+--echo # end of 10.8 test
+
+--disable_query_log
+set global innodb_buffer_pool_size = @save_size;
+--enable_query_log
```

Comment:
There is no need to disable the query log here. If you had used `let` and `eval` then the equivalent of `@save_size` would have been replaced in the `.result` file.